### ISSUE TITLE
Get 401 when calling next_uri

### ISSUE THREAD

**Issue by ThomsCass** [Lendix/bankin#1]
Hey,

Thank you for this awesome module !

When I have a pagination in the result of `transactions.list`

Here is the `pagination` object : 
```javascript
:{
"previous_uri": null,
"next_uri": "/v2/transactions?after=MjAxNi0xMi0yNzozODAwMDAwMzgyMTMwMA%3D%3D&limit=50"
}
```

When I call next() the request crash and I got a 401 error...

Thank you for your help !

**Comment by PierrickP** [comment]
Hi,

I checked and fixed on version `0.0.5`
The `%3D%3D` is not a problem. We forget to pass the auth token to the next/previous request.

ref: cebdac86fd4f1e102ae318f9695e019595bfaeba

Thanks for you issue.

**Comment by ThomsCass** [comment]
Are you sure its working ? I'm still getting the issue ...

**Comment by PierrickP** [comment]
@ThomsCass i tried again with the updated `0.0.5` version. It's working on `transactions` with  `next()`

+ Are you sure you using the new version?
+ Can you get the full request ? (should have the `Authorization` header)

**Comment by ThomsCass** [comment]
I'm using the last version ! I uninstall and re install to be sure.

I can't get it all I got is the Error with 401. But before calling the next() method it works well and I have the `auth` parameter

```javascript
Client {
endpoint: 'https://sync.bankin.com/v2',
clientId: '531af0294dc24685af1e496',
clientSecret: 'EDITED',
version: '2016-01-18' 
},
auth: 'b2f05245a027b64c7a3d64fdf5bbc0517e16477d-'
```

**Comment by PierrickP** [comment]
grumpf, working for the first `next()` but not the second ...
I'm fixing

**Comment by PierrickP** [comment]
Ok fixed `0.0.6`,
You can test with https://gist.github.com/PierrickP/71078ad2c6e6907cb4842e59d6bd00ed

+ Open an ngrok tunnel
+ Fill `NGROK` `BANKIN_SANDBOX_CLIENT_ID` et `BANKIN_SANDBOX_CLIENT_SECRET` (use sandbox id)
+ run
+ Open the printed link
+ Use `123456789` `demo0` (https://apidocs.bankin.com/v2/docs/testing-the-api)
+ Wait a bit. It ll print 4 transaction (3 `next`, 1 `previous`)

**Comment by ThomsCass** [comment]
It's working ! :)
